content/page/privacy-policy.en.md

```diff
@@ -16,7 +16,11 @@ The application requires the following permissions:
 * *QUERY_ALL_PACKAGES*: This permission allows the application to see all installed applications on your device.
 * *POST_NOTIFICATIONS*: This permission allows the application to post notifications. **This permission is only used on Android 13 and above.**
 
-The application transmits the list of installed applications on the device. This list is not kept by the εxodus server.
+We use PackageManager API to get applications handle (ID of app) from all apps installed (applications installed by the user and enabled on the device).
```

Comment:
Ok to add not disabled by the user
We have decide to not analyze disabled applications because disabled applications can't do anything on the device.